internal/singleprogram/singleprogram.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/sourcegraph/sourcegraph/internal/version"
 )
 
+const AppDirectory = "sourcegraph"
```

Comment:
```suggestion
const appDirectory = "sourcegraph"
```
I don't think this needs to be exported